runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -489,3 +489,30 @@ PROTO_CMD?=protoc
 SOLINK_SLINK+=protobuf ssl
 
 CXX_DEFINES+=GOOGLE_PROTOBUF_NO_RTTI
+
+ifneq ($(OPENSSL_CFLAGS),)
+    SET_FLAG=false
+    # --with-openssl=system, OPENSSL_LIBS is -L/usr/local/lib -lssl -lcrypto
```

Comment:
Can we change the `-L/usr/local/lib` to `-L/path/to/system/openssl` or something like that? Depending on the distro it can be `/usr/lib`, `/usr/local/lib`, `/usr/lib64`, `/lib`, etc or a completely different path, so mentioning a specific location can mislead.